src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,72 @@
+package duke;
+
+import duke.common.Const;
+
+/**
+ * This class handles errors in Duke.
+ */
+public class DukeException extends Exception {
```

Comment:
I like how the exception handling is done! 👍 Does this mean that your program will only ever throw exceptions extending from this class?

Reply:
Yes. I think it handles mainly the user input.